collector/logical_disk.go

```diff
@@ -250,6 +277,27 @@ func (c *LogicalDiskCollector) collect(ch chan<- prometheus.Metric) (*prometheus
 			float64(volume.SplitIOPerSec),
 			volume.Name,
 		)
+
+		ch <- prometheus.MustNewConstMetric(
+			c.ReadLatency,
+			prometheus.CounterValue,
+			float64(volume.AvgDiskSecPerRead),
```

Comment:
These values are probably actually the total number of seconds spent on reads. For example, on my system, I have `AvgDisksecPerRead : 1561802799`

Reply:
Hi @carlpett ,

Thank you for looking the code.
this value (AvgDisksecPerRead : 1561802799) represent the raw data sample... 
you should calculate the interval between samples, then calculate the difference to get an "average over time period".

Thanks
Moshe

Reply:
Yes. I think what I meant (and I wasn't very clear, sorry about that) is that we should probably rename the metric to reflect what the value actually is. So `read_latency_seconds_total`, and so on, to match the upstream best practice naming style.

Reply:
can you please get this MSI which has new logical disk perf metrics is added

Reply:
Please do not nag like this. I have updated your PR with a comment on what you need to do to get it merged.